importer/models.py

```diff
@@ -21,3 +25,12 @@ def delete_dynamic_model(instance, sender, **kwargs):
         # Removing Field Schema
     except Exception as e:
         logger.error(f"Error during deletion of Dynamic Model schema: {e.args[0]}")
+
+
+class ResourceHandler(models.Model):
+
+    """
+    Here we save the relation between the geonode resource created and the handler that created that resource
+    """
+    resource = models.ForeignKey(ResourceBase, blank=False, null=False, on_delete=models.CASCADE)
+    module_path = models.CharField(max_length=250, blank=False, null=False)
```

Comment:
Moving the class name to something generic, this field name needs to be more specific, like `handler_module_path`